src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/DOMMessages.properties

```diff
@@ -21,28 +21,28 @@
 # The messages are arranged in key and value tuples in a ListResourceBundle.
 
         BadMessageKey = The error message corresponding to the message key can not be found.
-        FormatFailed = An internal error occurred while formatting the following message:\n  
```

Comment:
At first glance, it might look like something should follow the `:`, but note that there is a `\n` so if anything this will only make the next line improperly indented.

Reply:
That could have been intentional though. Not sure if it was a good idea, but indenting something is a way of making something stand out as a quote. But looking at every use site, there is an extra space added anyway, so this seems fine. If we wanted to preserve the exact same behavior, all use sites should be updated to add 3 spaces.